TableApplication/Feature/FeatureListViewController.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+class HomeScreenViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    var homeScreenTableView = UITableView(frame: .zero, style: .plain)
+    var temporaryLabel = UILabel(frame: CGRect(x: 0, y: 0, width: 300, height: 15))
+    let identifier = "MyCell"
+    var features = [Feature(name: "List of Employees", task: "Show list of employees", id: UUID().uuidString)]
+    
+    private struct constants {
+        static let heightOfRow : Float = 80.0
+        static let titleOfView : String = "List of Features"
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = constants.titleOfView
+        createTable()
+        view.addSubview(homeScreenTableView)
+        temporaryLabel.center = CGPoint(x: 170, y: 150)
+        temporaryLabel.textAlignment = .natural
+        temporaryLabel.font = UIFont(name: "San Francisco", size: 5.0)
+        temporaryLabel.text = features.first?.task
```

Comment:
separate binding from layout

Reply:
Fixed

Reply:
extract temporaryLabel.text = features.first?.task to method